### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cathelest**
Please review changes to readme,

Feel free to scrap it if its not right, or if iv presumed something that may be beyond my particular setup and use case.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ homeassistant:
 If it doesn't already exist, create a "customize.yaml" file in the same directory as configuration.yaml and add the following:
 ```yaml
 sensor.fronius_total_energy:
-  state_class: measurement
+  state_class: total_increasing
```
Comment (colwilliamsnz) [inline README.md:34]: This customize.yaml detail is intended for people running HA 2021.8.x that have upgraded the Fronius component to 0.9.7. It is not required for HA 2021.9.x (check the readme).

File: README.md [hunk 2]
```diff
@@ -130,7 +133,7 @@ variable | required | type | default | description
...
 ``scan_interval`` | no | string | 60 | The interval to query the Fronius Inverter for data.
 ``powerflow`` | no | boolean | ``False`` | Set to ``True`` if you have a PowerFlow meter (SmartMeter) to add ``grid_usage``, ``house_load``, ``panel_status``, ``rel_autonomy`` and ``rel_selfconsumption`` sensors.
-``smartmeter`` | no | boolean | ``False`` | Set to ``True`` if you have a SmartMeter to add ``smartmeter_current_ac_phase_one``, ``smartmeter_current_ac_phase_two``, ``smartmeter_current_ac_phase_three``, ``smartmeter_voltage_ac_phase_one``, ``smartmeter_voltage_ac_phase_two``, ``smartmeter_voltage_ac_phase_three``, ``smartmeter_energy_ac_consumed`` and ``smartmeter_energy_ac_sold`` sensors.
+``smartmeter`` | no | boolean | ``False`` | Set to ``True`` if you have a SmartMeter to add ``smartmeter_energy_ac_consumed``,``smartmeter_energy_ac_sold``,``smartmeter_current_ac_phase_one``, ``smartmeter_current_ac_phase_two``, ``smartmeter_current_ac_phase_three``, ``smartmeter_voltage_ac_phase_one``, ``smartmeter_voltage_ac_phase_two``, ``smartmeter_voltage_ac_phase_three``, ``smartmeter_energy_ac_consumed`` and ``smartmeter_energy_ac_sold`` sensors.
```
Comment (colwilliamsnz) [inline README.md:136]: You have duplicated "smartmeter_energy_ac_consumed" and "smartmeter_energy_ac_sold"

File: README.md [hunk 3]
```diff
@@ -49,8 +49,8 @@ Where "fronius" in the sensor name needs to match the name of the integration. O
 The following "lifetime" sensors can be added to the energy configuration:
 
 * Solar production: ``total_energy``
-* Grid consumption: ``smartmeter_energy_ac_consumed``
-* Grid feed-in: ``smartmeter_energy_ac_sold``
+* Grid consumption: ``smartmeter_energy_ac_consumed`` (smartmeter required).
```
Comment (colwilliamsnz) [inline README.md:52]: Agree with this as it provides additional clarification.

File: README.md (hunk 2 above)
Comment (cathelest) [reply README.md:136]: Sorry, didnt see them on the list when i was editing it at midnight lol, sorry about that.

File: README.md (hunk 1 above)
Comment (cathelest) [reply README.md:34]: Ah yeah i think when i was trying to workout why the smartmeter entities werent showing up i must
have added the old measurement state classes to my customize.yaml, which then rightly through enexpected
state class errors on the energy dashboard, and i should have removed rather then changing to total increasing which
ofcorse wasnt neccesary, my applogise,

**Pull request closed without merging**